department-of-veterans-affairs/abd-vro
# Update package versions to fix failing build docker images.

Update package versions in Dockerfiles for app and pdfgenerator to fix failing build docker images.

## What was the problem?
failed to build docker images as the package versions used are no longer available.


## How does this fix it?[^1]
Update the versions.

## How to test this PR
- Step 1. Build the docker images successfully.
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
